### ISSUE TITLE
Avoid selection of runs with different granularity and epoch length

### ISSUE THREAD

**Issue by thinkh** [Caleydo/malevo#145]
With the revised timeline layout #126 using a single timeline, we introduced the following constraints:

> * All runs have same epoch length
> * The granularity of all runs is equal

To meet this requirements we have to extend the run selector. After the user selects the first run all incompatible runs (i.e., different epoch length and granularity) are disabled in the selector. This way we prevent mixing incompatible runs.

**Comment by NMO13** [comment]
I don't think we have to check these requirements in malevo. I would say that this is a data requirement that the provider of the runs has to meet.

**Comment by gfrogat** [comment]
Malevo should only show all the relevant runs which can be compared. It's a simple filtering functionality that avoids errors due to incompatible granularities. If we don't want the user to encounter this type of error we simply only show compatible runs in the selection window.

Maybe you have a better idea how to solve this but we should not require the user to handle this.

**Comment by mstreit** [comment]
I think the solution to filter the runs based on the first selection is simple and effective. This way we can handle multiple compatible run sets in our interface without letting the user run into compatibility problems.
